### ISSUE TITLE
2.9.1 - Avast detection report - false positive?

### ISSUE THREAD

**Issue by rafi-d** [phw198/OutlookGoogleCalendarSync#1296]
Attaching the screenshot... 
![1](https://user-images.githubusercontent.com/2458280/134625741-e6f44695-0f76-44d2-aca1-79f837774e80.png)

**Comment by euinor** [comment]
I'm getting a warning too, with Symantec Antivirus. Unfortunately as it is a work machine I am blocked from permanently adding the file to the exceptions list. I can however just click past the warning.
Edit: The malware detected is 'Left Alone' WS.Reputation.1. This is with the 2.9.1. Alpha Portable version. The previous Beta version did not trigger this detection
![image](https://user-images.githubusercontent.com/53984070/134633567-716881ed-86db-4653-a42d-9048bfb9a117.png)

**Comment by phw198** [comment]
Thanks @rafi-d, I've sent this info onto Avast support for them to whitelist.

@euinor Looks like symantec just needs some time to build trust on the file - hopefully will stop warning soon.

NB: There is a [wiki entry regarding false positives](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Known-Issues#anti-virus-false-positives) and links to submission forms to report them.

**Comment by euinor** [comment]
thanks for the form links. I submitted the file as a false positive, as while Symantec gave me a warning, it sent IT a detection...

**Comment by phw198** [comment]
@rafi-d Avast support say they are unable to reproduce the false positive for the 2.9.1 setup executable. 

If it is still happening for you (make sure you have the latest virus definitions downloaded), could you provide the screenshot, but make sure you include the alert ID which appears below the "Status" line at the bottom.

**Comment by rafi-d** [comment]
I did add here a screenshot already. As you can see, I have excluded it in Avast, so no issues any more, but this is not a real solution...  :) 
Anyways, I've removed the exclusion , and will see what happens...

**Comment by phw198** [comment]
Yes, but your screenshot has cropped out the alert ID, which is what Avast support require.
